airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -373,8 +373,8 @@ def sync(self) -> None:
                         self.result_queue.put(results)
                 finally:
                     self.result_queue.task_done()
-            except Empty:
-                break
+        except Empty:
+            pass
```

Comment:
Use `suppress`?

Reply:
Yes! But I reverted the changes in this file, because the PR should change only GCP provider.